docs/proposals/scheduling/20240201-enbale-reservation-preempt.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Enable-Reservation-Preempt
+authors:
+- "@xulinfei1996"
+reviewers:
+- "@buptcozy"
+- "@eahydra"
+- "@hormes"
+creation-date: 2024-02-01
+last-updated: 2024-02-01
+status: provisional
+
+---
+
+# Enable Reservation Preempt
+
+<!-- TOC -->
+
+- [Enable Reservation Preempt](#enable-reservation-preempt)
+    - [Summary](#summary)
+    - [Motivation](#motivation)
+        - [Goals](#goals)
+        - [Non-goals/Future work](#non-goalsfuture-work)
+    - [Proposal](#proposal)
+        - [Key Concept/User Stories](#key-conceptuser-stories)
+        - [Implementation Details](#implementation-details)
+            - [Priority](#priority)
+            - [Preemption](#preemption)
+                - [Response To Preempted](#response-to-preempted)
+                - [Extension Point](#extension-point)
+                - [Over All](#over-all)
+                - [PreFilter](#prefilter)
+                - [PostFilter](#postfilter)
+        - [API](#api)
+            - [Reservation](#reservation)
+        - [Compatibility](#compatibility)
+    - [Unsolved Problems](#unsolved-problems)
+    - [Alternatives](#alternatives)
+    - [Implementation History](#implementation-history)
+    - [References](#references)
+
+<!-- /TOC -->
+
+## Summary
+This proposal provides reservation preempt mechanism for the scheduler.
+
+## Motivation
+In business scenarios, there may be excessive creation of reservations. In such cases, reservations may not be scheduled
+due to insufficient resources. What's more, reservations can set different priority to declare different level SLA resources.
+But reservation preemption is not supported now, so users may hope to support reservation preemption.
+Additionally, reservations may be created by batch, so it is expected that reservation preemption can support batch preemption.
+
+### Goals
+
+1. Define API to announce reservation can trigger preemption and can be preempted.
+
+2. Define API to set reservation priority.
+
+### Non-goals/Future work
+
+## Proposal
+
+### Key Concept\User Stories
+Story 1: Reservation failed to schedule due to insufficient cluster resource, so need to preempt other reservations.
```

Comment:
This information is still not broken down. According to the previous comment, I will continue to disassemble it.